src/i18n/en.js

```diff
@@ -288,38 +261,25 @@ export default {
     label_lastname: `Lastname: `,
     label_password: `Password: `,
     label_phone: `Phone: `,
-    legal_HTML: `
-      <p>
-        Doppler informs you that the personal data you provide by completing this form will be
-        treated by Doppler LLC as responsible for this web site.
-      </p>
-      <p>
-        <strong>Purpose:</strong> Sign you up into our platform and provide the services that you require.
-      </p>
-      <p>
-        <strong>Legitimation:</strong> Consent of the applicant.
-      </p>
-      <p>
-        <strong>Recipients:</strong> Your data will be saved by Doppler, Zoho as CRM, Google as the
-        provider of reCAPTCHA service, Digital Ocean, Cogeco Peer1 and Rackspace as hosting companies.
-      </p>
-      <p>
-        <strong>Additional information:</strong> In Doppler's
-        <a target="_blank" href="${urlPrivacyFromSignup_HTMLEncoded}" ${patchForBlank}>Privacy Policy</a>
-        you'll find additional information about the data storage and use of your
-        personal information, including information on access, conservation, rectification,
-        deletion, security, cross-border data transfers and other issues.
-      </p>
-      `,
+    legal_MD: `Doppler informs you that the personal data you provide by completing this form will be treated by Doppler LLC as responsible for this web site.
+
+**Purpose:** Sign you up into our platform and provide the services that you require.
```

Comment:
What do you think about always start long (more than a line) contents with a line before?

```suggestion
    legal_MD: `
Doppler informs you that the personal data you provide by completing this form will be treated by Doppler LLC as responsible for this web site.

**Purpose:** Sign you up into our platform and provide the services that you require.
```

Reply:
yes i changed all now